sdk/search/search-documents/.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": ["../../.eslintrc.json"],
+  "rules": {
+    "@azure/azure-sdk/ts-package-json-engine-is-present": "warn"
```

Comment:
The codebase at this point in time required `engines.node` to be set to >= 8.0.0, and I didn't want to start shaving yaks by changing the rule. So I think warn is ok, but let me know if you disagree

Reply:
In that case lets skip the change to the engines field. The next 11.3.0 will have that update anyway and we are doing our due diligence in the readme file around documenting the re-requisites